indico/modules/users/tasks.py

```diff
@@ -37,3 +37,8 @@ def update_gravatars(user=None):
             continue
         set_user_avatar(user, gravatar, 'gravatar', lastmod)
         logger.info('Gravatar of user %s updated', user)
+
+
+# Should be similar to the Material export
```

Comment:
I think you can easily ditch polling etc like we have for the material export and just send an email to the user once the export is ready...